mappings/base_mapping.py

```diff
@@ -29,6 +29,22 @@ def applyMapping(self):
 
     def applyFormatting(self):
         return self.record
+    
+    def format_identifiers(self, identifiers, source):
+        identifier_type_dict = {8: 'issn', 10: 'isbn', 13: 'isbn'}
+        for index, identifier in enumerate(identifiers):
```

Comment:
I recommend using the `map` function for these cases where we are:
- applying a function to every item in a list
- returning the result as a list

Reply:
Sounds like a good idea